### ISSUE TITLE
Crashing 1.16.5

### ISSUE THREAD

**Issue by Livedeqq** [ViaVersion/ViaForge#94]
### General Information

ViaForge only

### Bug Description

Starting game and crash, i tried other forge versions and still crashing
[crash-2024-06-14_17.16.37-fml.txt](https://github.com/user-attachments/files/15840075/crash-2024-06-14_17.16.37-fml.txt)
[latest.log](https://github.com/user-attachments/files/15840082/latest.log)

### Steps to Reproduce

1. start game and game should crash 

### Expected Behavior

crash

**Comment by FlorianMichael** [comment]
Are you sure you have only ViaForge installed? Also try dev builds at https://ci.viaversion.com/view/Platforms/job/ViaForge/

**Comment by Darker935** [comment]
Me2, and the same crash related to slf4j

i tried the DEV version and the Release version

**Comment by Darker935** [comment]
> Are you sure you have only ViaForge installed? Also try dev builds at https://ci.viaversion.com/view/Platforms/job/ViaForge/

And im sure about that too

**Comment by Lingye1204** [comment]
i meet the same problem

**Comment by lunarydess** [comment]
@Livedeqq @Lingye1204 @Darker935 
try a newer java version than java 8
or atleast something newer than 1.8.0_51
i recommend java 17 or if you want to use a newer forge-loader (e.g. 36.2.42) java 21
for me both java 1.8.412-b08 and java 17.0.12 work

**Comment by lunarydess** [comment]
*not reproducible yet*

**Comment by Lingye1204** [comment]
> 尝试比 Java 8 更新的 Java 版本，或者至少比 1.8.0_51 更新的版本，我推荐 Java 17，或者如果您想使用更新的 forge-loader（例如 36.2.42）java 21，对我来说，java 1.8.412-b08 和 java 17.0.12 都可以

yes,i crashed for using java 8 with force selection by the launcher.but the launcher didnt remind me that i should change a java

**Comment by AlphaGameDeveloper** [comment]
Crash - 1.20.1 with ViaForge

**Comment by BUCKQIAN** [comment]
I am using other Java versions (such as JDK 17) and cannot run them. Error message displayed: ViaForge (viaforge) encountered an error during the common_stup event phase java. 1ang NoC1assDefFoundError: org/s1f4j/spi/SLF4JServiceProvider
What's going on here?(Minecraft 1.16.5)

**Comment by 10xA** [comment]
> I am using other Java versions (such as JDK 17) and cannot run them. Error message displayed: ViaForge (viaforge) encountered an error during the common_stup event phase java. 1ang NoC1assDefFoundError: org/s1f4j/spi/SLF4JServiceProvider What's going on here?(Minecraft 1.16.5)

I am able to replicate this as well

**Comment by HowardZHY** [comment]
If this still happens with 3.8.0, try to add 0 at the start of the mod file name to make it has priority when loaded by Forge

**Comment by HowardZHY** [comment]
Seems SLF4J Hates Java 8u51, try to use something like 8u311

**Comment by Developer-Major-Cozmic** [comment]
So what's the solution?

**Comment by FlorianMichael** [comment]
Try what @HowardZHY wrote, otherwise there is none at the moment.

**Comment by MochisDrak** [comment]
Same Issue here, tried it all as a smoothbrained mc player but still it does not work 😭

**Comment by joeyjumper94** [comment]
also affects the 1.18.2 version

**Comment by ILikeCodingg5565** [comment]
is there a solution to this yet?
